code/__DEFINES/layers.dm

```diff
@@ -13,29 +13,30 @@
 #define PLANE_SPACE -21
 #define PLANE_SPACE_PARALLAX -20
 
-#define GRAVITY_PULSE_PLANE -12
+#define GRAVITY_PULSE_PLANE -19
 #define GRAVITY_PULSE_RENDER_TARGET "*GRAVPULSE_RENDER_TARGET"
 
-#define RENDER_PLANE_TRANSPARENT -11 //Transparent plane that shows openspace underneath the floor
+#define RENDER_PLANE_TRANSPARENT -18 //Transparent plane that shows openspace underneath the floor
+#define TRANSPARENT_FLOOR_PLANE -13
 
-#define TRANSPARENT_FLOOR_PLANE -10
+#define FLOOR_PLANE -12
+#define FLOOR_PLANE_RENDER_TARGET "*FLOOR_PLANE"
+#define WALL_PLANE -11
+#define GAME_PLANE -10
+#define ABOVE_GAME_PLANE -9
 
-#define FLOOR_PLANE -6
-
-#define WALL_PLANE -5
-#define GAME_PLANE -4
-#define ABOVE_GAME_PLANE -3
```

Comment:
there's no reason to shift these over if we're not adding new planes, should just be as they are now